### PR TITLE
do not jump to the quickfix window when metalint

### PR DESCRIPTION
**Pull request by jinleileiking**
#1290 

copy codes from `ale`

### CODE REVIEW

File: autoload/go/lint.vim [hunk 1]
```diff
@@ -255,9 +255,15 @@ function s:lint_job(args)
...
-    " TODO(arslan): cursor still jumps to first error even If I don't want
-    " it. Seems like there is a regression somewhere, but not sure where.
+    " TODO(jinleileiking): give a configure to jump or not
+    let l:winnr = winnr()
+
     copen
+
+    " If focus changed, restore it (jump to the last window).
+    if l:winnr !=# winnr()
+      wincmd p
```
Comment (fatih) [inline autoload/go/lint.vim]: I think this is better:

```
exe l:winnr . "wincmd w"
```

Jumps to the saved windows directly

Comment (jinleileiking) [inline autoload/go/lint.vim]: I am not very god at VIML. So I follow your orders :)

Comment (fatih) [inline autoload/go/lint.vim]: Yeah I've tested it. If you change it I'm going to merge it.

Comment (jinleileiking) [inline autoload/go/lint.vim]: ok, I will modify

Comment (jinleileiking) [inline autoload/go/lint.vim]: I have changed the codes, and test on my machine.

**Comment by fatih** [comment]
Thanks @jinleileiking 👍

**Pull request merged**